### PR TITLE
Question 3 finished, tested with sample code commented out at the bot…

### PR DESCRIPTION
**Pull request by mattb61**
Honestly not sure if animInfo method is needed but everything else should work just fine.

### CODE REVIEW

File: Assignment_2/Question_3.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+class Dog(Animal):
+
+
+    def __init__(self, name="", species="", breed=""):
+        Animal.__init__(self, name, species)
+        self.breed = breed
+
+    # def breed(self, dogBreed):
+    #     self.dogBreed = dogBreed
```
Comment (Alexander-Moyles) [inline Assignment_2/Question_3.py:27]: What are these commented lines for?

Comment (mattb61) [reply Assignment_2/Question_3.py:27]: They were originally going to be a property for the dogBreed attribute, but I realized afterwards that it wasn't needed and that I didn't even know what I was doing with it lol. I more than likely forgot to delete them after finishing the class.

**Comment by Alexander-Moyles** [comment]
Hey, this request is still going to the master branch, you need to pull it to the Assignment2 branch instead

**Comment by mattb61** [comment]
My bad, forgot all about it due to the test and other assignment. Just did it.

**Comment by ChristopherSquires** [comment]
I think I've pushed or merged or whatever the hell it's called question 4 and it's classes into the repository. I recommend double checking if I put the question and it's classes where it need to be though because this makes me want to slam my head into a wall

**Pull request merged**